watercore-api/src/main/java/me/lucyy/watercore/api/version/SemanticVersion.java

```diff
@@ -63,20 +71,25 @@ public int getPatch() {
 	 * including a hyphen in theversion.
```

Comment:
is that a typo i see :eyes:

Reply:
Not as bad as the spigot potioneffecttype docs xD
(see pic attached)
<img width="732" alt="Capture d’écran 2021-05-03 à 20 18 55" src="https://user-images.githubusercontent.com/76099886/116928637-dbcbf180-ac4c-11eb-928c-7ed7eebf5d72.png">